examples/full-example/app.js

```diff
@@ -33,7 +33,7 @@ app.get('/', function(req, res) {
             // WARNING: This temporary "lazy" bundle is not part of the example
             // The dev team needs this here for a while in order to
             // do some work in parallel branches
-            bundle(req, 'temporary', variations),
+            entryMap(req, 'temporary', variations),
```

Comment:
For example 
`
entryMap(req, 'temporary', variations)
entryMap(req, 'temporary2', variations)
entryMap(req, 'temporary3', variations)
`
So we need to call `req.mendel.getBundleEntries()` and `findTreeForVariations` and `immediate function` 3 times right?